mne/channels/channels.py

```diff
@@ -284,6 +284,20 @@ def rename_channels(self, mapping):
         """
         rename_channels(self.info, mapping)
 
+    def set_montage(self, montage):
+        """Set EEG sensor configuration
+
+        Note. Operates in place.
+
+        Parameters
+        ----------
+        montage : instance of mne.channels.Montage or
+                  mne.io.meas_info.DigMontage
+            The MNE montage object.
```

Comment:
Have you checked how this renders in doc? It might have to be on one line, in which case you can add `# noqa` after the `"""` end-quote if the line ends up too long. Also, please add a `Notes` with a `.. versionadded` directive (see existing examples).